### ISSUE TITLE
Standing Agenda, Public Core Meeting Meeting - January 2017

### ISSUE THREAD

**Issue by gundalow** [ansible/community#148]
Please leave a comment regarding any agenda item you wish to discuss. If you don't show up for the meeting, your item will be skipped.

If your IRC nick is different from your Github username, leave that as well.

See https://github.com/ansible/community/blob/master/MEETINGS.md for the schedule

Once an item has been addressed it should get ~~strike-though~~ `~~strike-though~~`

**Comment by gundalow** [comment]
Re-migrated ansible/ansible#19016
* From 12/13 meeting: currently waiting for JMainguy to have a chance to look.

**Comment by gundalow** [comment]
Re-migrated https://github.com/ansible/ansible/pull/19020

* From today's meeting, ryansb has this one on his list.

**Comment by gundalow** [comment]
Unfortunately when I migrated he first time the original was deleted so I've had to create a new PR. https://github.com/ansible/ansible/pull/19021
* From 12/15 meeting: This one is waiting on the submitter to make some changes.

**Comment by gundalow** [comment]
Re-migrated https://github.com/ansible/ansible/pull/19027
* From 12/15 meeting: abadger will review with some questions from bcoca's PR.

**Comment by gundalow** [comment]
Re-migrated https://github.com/ansible/ansible/pull/19042 [postgresql module] <strike>and https://github.com/ansible/ansible/pull/19043 [elasticache module]</strike>

* Elasticache module merged by ryansb.  abadger looking over postgresql module

**Comment by gundalow** [comment]
Re-migrated PR https://github.com/ansible/ansible/pull/19049
* Asking dav1x to review this one and decide between the two alternative implementations.

**Comment by gundalow** [comment]
Re-moved ansible/ansible#19070
* nitzmahone has this on his list.

**Comment by jasperla** [comment]
As mentioned in https://github.com/ansible/ansible/pull/18924 , I've now re-submitted the PR as https://github.com/ansible/ansible/pull/19680 .

**Comment by bcoca** [comment]
https://github.com/ansible/ansible/pull/19664

**Comment by bcoca** [comment]
https://github.com/ansible/ansible/pull/19800

**Comment by gundalow** [comment]
https://github.com/ansible/ansible/issues/15432

**Comment by mattclay** [comment]
https://github.com/ansible/ansible/pull/18147

**Comment by mattclay** [comment]
Discuss extending the module "shipit" workflow to non-modules, such as:

- lib/ansible/plugins
- lib/ansible/module_utils
- contrib/inventory

The bot would need to be updated to support the updated workflow.

**Comment by gundalow** [comment]
mount module feature request: allow to mount without adding to fstab https://github.com/ansible/ansible/issues/19820

**Comment by gundalow** [comment]
https://github.com/ansible/ansible/pull/19500 Update assemble to allow alternate source of files

**Comment by podlesh** [comment]
feature proposal: https://github.com/ansible/ansible/pull/19707

**Comment by jbaublitz** [comment]
feature proposal: Add [`google-api-python-client`](https://developers.google.com/api-client-library/python/) as a supported alternate back end for existing `gce*` modules alongside `libcloud`.

@ryansb: In case I don't make the meeting here is a description of the problem

There is a potential race condition in `libcloud` when deleting a subnetwork then a network from [`gce_net`](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/google/gce_net.py). This seems to stem from [`ex_destroy_subnetwork`](https://github.com/apache/libcloud/blob/trunk/libcloud/compute/drivers/gce.py#L6859) in `libcloud` and specifically [this call](https://github.com/apache/libcloud/blob/trunk/libcloud/compute/drivers/gce.py#L6903). `request` in `libcloud` connection objects assume that the API is synchronous and if subnetwork deletion on Google is an async API (I'm assuming it is but @supertom can confirm this), the `gce_net` subnetwork deletion would "succeed" according to `libcloud` but would not be complete by the time ansible would start trying to delete the network. The network deletion would subsequently fail because the subnetwork would not be deleted yet. This is the behavior that I'm seeing and will work with `libcloud` to patch this but would also propose using the Google library as an alternate backend. As a side note, I've worked with both the library and some of the `gce` modules so I'm willing to do the alternate back end implementations for the modules I've been contributing to and potentially others when time permits.

**Comment by bcoca** [comment]
https://github.com/ansible/ansible/pull/20004 cc @evgeni

**Comment by evgeni** [comment]
IRC nick: Zhenech

**Comment by abadger** [comment]
SmartOS Timezone module: https://github.com/ansible/ansible/pull/20105  Do we want this as a separate module or merged with the existing timezone module?

**Comment by jasperla** [comment]
For your consideration in addition to @abadger 's comment, please note that other systems (e.g. OpenBSD) are not supported by the existing `timezone` module. For those systems it's merely a matter of reading/setting the `/etc/localtime` symlink and all the extras that `timezone` currently provides won't be used.

**Comment by bcoca** [comment]
https://github.com/ansible/ansible/pull/18643

**Comment by abadger** [comment]
From @jtyr: https://github.com/ansible/ansible/pull/19283  hosts module (for /etc/hosts)

**Comment by abadger** [comment]
From  luto: https://github.com/ansible/ansible/pull/19297 Fix for wildcards inside of a path for fileglob lookup (ie: ```with_fileglob: "/tmp/*/some.conf"```)

**Comment by bcoca** [comment]
Decide which are valid directives on include (for execution or inheritance) and what to do when we decide they are not.

**Comment by rfleschenberg** [comment]
* Verisioned docs / readthedocs
  * Versioned docs would be great! Notes in the docs about which versions they apply to are good, but the default docs not applying to the default (released) Ansible version is a major UX problem.
  People skim through docs, and they are right to do so. Time constraints are a thing. Looking up 
  module parameter names in the official docs to find out that the official release does not support
  them is no fun, especially not for newcomers.
  * People (at least Python people) are used to RTD
  * Ansible can build upon the experiences of other projects (e.g. Django)
  * Supporting RTD would be cool (I'm not affiliated with them)
  * irc nick: moldy

**Comment by gundalow** [comment]
https://github.com/ansible/ansible/pull/20220/files module_utils/basic.py: Support logical or condition in required_if

**Comment by gundalow** [comment]
https://github.com/ansible/ansible/pull/20673 Refurbish developing modules content - stage 1

**Comment by allanice001** [comment]
ansible/ansible/#20737 Bugfix for ansible/hacking/test-module

**Comment by bcoca** [comment]
https://github.com/ansible/ansible/pull/20058

**Comment by gundalow** [comment]
Orphaned mode process (uri)
https://github.com/ansible/ansibullbot/pull/291

**Comment by willthames** [comment]
I would like to discuss custodianship of the AWS modules. In particular I would like:

* extra people with shipit
* extra people with commit rights
* someone to review my open PRs, many of which have been moved after months languishing in ansible-modules-core or ansible-modules-extras. The original authors have in some cases completely given up on the project, but their contributions are still valuable.

**Comment by bcoca** [comment]
https://github.com/ansible/ansible/pull/18379

**Comment by gundalow** [comment]
Done
Feb 2017 agenda can be found at https://github.com/ansible/community/issues/150
